### PR TITLE
Use sequence number instead of p-number

### PR DESCRIPTION
**Pull request by NicklasHugoy**

### CODE REVIEW

File: data_handler.py [hunk 1]
```diff
@@ -246,7 +246,8 @@ def filter_industry_codes(data: dict) -> bool:
         Checks if row 'data' has a valid industry code.
         """
         include_codes = ['561010', '561020', '563000']
-        res = 'industry_code' in data.keys() and data['industry_code'] in include_codes
+        res = 'industry_code' in data.keys(
+        ) and data['industry_code'] in include_codes
```
Comment (Badgie) [inline data_handler.py]: I think your linter is set to 80 char line length. I'd prefer 100.

File: data_handler.py [hunk 2]
```diff
@@ -255,7 +256,8 @@ def filter_null_control(data: dict) -> bool:
         """
         Checks if row 'data' has received at least 1 control check.
         """
-        res = 'smiley_reports' in data.keys() and len(data['smiley_reports']) > 0
+        res = 'smiley_reports' in data.keys() and len(
+            data['smiley_reports']) > 0
```
Comment (Badgie) [inline data_handler.py]: Same as the other comment.

**Review by Badgie — APPROVED** [review]
Some linting stuff otherwise :shipit:

File: data_handler.py (hunk 1 above)
Comment (NicklasHugoy) [reply data_handler.py]: How to change? Flake8 on vscode

Comment (Badgie) [reply data_handler.py]: Set `max-line-length = 100` in your flake8 config. Should be in `C:\Users\<username>\.flake8`

**Pull request merged**